src/services/room-service.spec.js

```diff
@@ -167,17 +173,26 @@ describe('room-service', () => {
     let myPrivateRoom = null;
 
     beforeEach(async () => {
+      userService.getUserById = sandbox.stub().resolves(myUser);
       [myPublicRoom, myPrivateRoom] = await Promise.all([
         await sut.createRoom({ name: 'my public room', access: ROOM_ACCESS_LEVEL.public, user: myUser }),
         await sut.createRoom({ name: 'my private room', access: ROOM_ACCESS_LEVEL.private, user: myUser })
       ]);
     });
 
+    afterEach(() => {
+      sandbox.restore();
+    });
+
     it('should create a new invitation if it does not exist', async () => {
       const { invitation } = await sut.createOrUpdateInvitation({ roomId: myPrivateRoom._id, email: 'invited-user@test.com', user: myUser });
       expect(invitation.token).toBeDefined();
     });
 
+    it('should throw a bad request if the owner invites herself', () => {
```

Comment:
`... if owner invites themselves`

Reply:
But themselves is plural. Herself is the new polite way to treat someone when you don't know the gender.

Reply:
"Themselves" can be used in the singular non-gender specific case: 
"Both “themselves” and “themself” are acceptable as reflexive singular pronouns; however, “themselves” is currently the more common usage."
"Herself" is feminine, as "himself" is masculine. But anyway, I was not into discussing genders or political correctness, I just wanted to point out that I find the phrasing towards a gender confusing and unnecessary.

Reply:
Maybe it got updated in the meantime, when I was writing/reviewing research papers this was the rule, but it's been a few years. Anyway, I'll adjust.